gorapass/views.py

```diff
@@ -47,24 +51,31 @@ def hike(request, hike_id):
 def hikes(request):
     hike_models = Hikes.objects.all()
 
-    # Filter out hikes if there is filtration criteria in the request
-    if request.body:
-        selectors = json.loads(request.body)['selectors']
-        valid_selectors_status = ModelFilter.validate_selectors(Hikes, selectors)
-        if not valid_selectors_status['success']:
-            return HttpResponseBadRequest(valid_selectors_status['message'])
+    if request.method == 'GET':
+        hike_dicts = [ model_to_dict(stamp) for stamp in hike_models ]
+    else:
```

Comment:
Should we hard code it so that it's `if request.method == 'POST'`? I'm not sure how careful we need to be with other request types, but might not be the worst idea to be explicit?

Reply:
Yeah that's a good idea.